communication/slack-config/channels.yaml

```diff
@@ -306,6 +306,7 @@ channels:
   - name: kwok-dev
   - name: kyverno
   - name: kyverno-dev
+  - name: kyverno-chainsaw
```

Comment:
The channel names need to be in alphabetical order, could you update this to be above `kyverno-dev`?

Reply:
@jeefy done.